src/Open3D/Visualization/Visualizer/GuiVisualizer.cpp

```diff
@@ -176,21 +226,6 @@ GuiVisualizer::GuiVisualizer(
     auto &app = gui::Application::GetInstance();
     auto &theme = GetTheme();
 
-    // Create material
-    std::string err;
-    std::string rsrcPath = app.GetResourcePath();
-    std::string path = rsrcPath + "/nonmetal.filamat";
-    impl_->nonmetal =
-            GetRenderer().AddMaterial(ResourceLoadRequest(path.data()));
-    impl_->white =
-            GetRenderer()
-                    .ModifyMaterial(impl_->nonmetal)
-                    .SetColor("baseColor", Eigen::Vector3f{1.0, 1.0, 1.0})
-                    .SetParameter("roughness", 0.5f)
-                    .SetParameter("clearCoat", 1.f)
-                    .SetParameter("clearCoatRoughness", 0.3f)
-                    .Finish();
```

Comment:
It's not clear from the Github review where the material creation went to, just noting that we'll need SetDoubleSided(true) there to make sure things like cactusgarden.ply, fountain.ply, and stonewall.ply work. You probably did that, in which case, resolve this comment.

Reply:
Yeap, it is gone `AddGeometry(geometry);` method. And I set `doubleSided` there.